### PR TITLE
fix: loading spinner text to be more readable, appSettings toggle overlap bug

### LINKED ISSUE #6663: Text is hard to read when creating a new bot
## Describe the bug

When creating a new bot the post selecting a template, you have a series of informational items that are gone through, however the text is really hard to read on top of the text of the existing screen.  See screenshot example below:

image

## Version

Version:  1.4.0-nightly.230410.31d4ca5
Electron: 8.2.4
Chrome: 80.0.3987.165
NodeJS: 12.13.0
V8: 8.0.426.27-electron.0

## Browser

- [x] Electron distribution
- [ ] Chrome
- [ ] Safari
- [ ] Firefox
- [ ] Edge

## OS

- [ ] macOS
- [x] Windows
- [ ] Ubuntu

## Expected behavior

Text on screen should be easily readable.  I would expect this to be in a box without transparent background to ensure that it is easily readable.

### LINKED ISSUE #6801: Toggle to see appsettings advanced view rendering issue
Toggle to see appsettings advanced view renders within the json viewer and leads to weird overlap with JSOn text

### PR DESCRIPTION
**Pull request by pavolum**
## Description

Small UI changes to resolve two UI bugs. 

First bug is that the loading text was hard to read during bot creation so that has been formatted to be rendered in its own modal similar to the spinner one sees when installing a new package

Second bug was the positioning of the toggle for advanced settings which overlapped the json editor and led to character overlap between the toggle and text in the JSON editor. 

## Task Item

fixes #6663 
fixes #6801 

## Screenshots

image

image

### CODE REVIEW

File: Composer/packages/client/src/router.tsx [hunk 1]
```diff
@@ -42,6 +43,11 @@ const BotCreationFlowRouter = React.lazy(() => import('./components/CreationFlow
 const BotCreationFlowRouterV2 = React.lazy(() => import('./components/CreationFlow/v2/CreationFlow'));
 const FormDialogPage = React.lazy(() => import('./pages/form-dialog/FormDialogPage'));
 
+const modalControlGroup = css`
+  border: 1px solid rgb(237, 235, 233);
+  padding: 0.5rem 1rem 1rem 1rem;
```
Comment (srinaath) [inline Composer/packages/client/src/router.tsx:48]: Do we use rems anywhere in code? Can we stick to pixels?

File: Composer/packages/client/src/router.tsx [hunk 2]
```diff
@@ -42,6 +43,11 @@ const BotCreationFlowRouter = React.lazy(() => import('./components/CreationFlow
 const BotCreationFlowRouterV2 = React.lazy(() => import('./components/CreationFlow/v2/CreationFlow'));
 const FormDialogPage = React.lazy(() => import('./pages/form-dialog/FormDialogPage'));
 
+const modalControlGroup = css`
+  border: 1px solid rgb(237, 235, 233);
```
Comment (srinaath) [inline Composer/packages/client/src/router.tsx:47]: Is there a matching border color from Shared, CommunicationColors?

Comment (pavolum) [reply Composer/packages/client/src/router.tsx]: Ive removed the border as it is not necessary

File: Composer/packages/client/src/router.tsx (hunk 1 above)
Comment (pavolum) [reply Composer/packages/client/src/router.tsx]: Changed to pixels!

**Review by srinaath — APPROVED** [review]

**Pull request merged**